libs/platform/src/lib/components/form/combobox/combobox-mobile/combobox/combobox-mobile.component.spec.ts

```diff
@@ -17,43 +17,37 @@ describe('ComboboxMobileComponent', () => {
         title: 'title', approveButtonText: 'approve', cancelButtonText: 'cancel', hasCloseButton: true
     };
 
-    let comboboxInputComponent: Partial<ComboboxComponent> = {
-        mobileConfig: {
+    class ComboboxInputComponent implements ComboboxInterface {
+        inputText: string;
+        openChange: Subject<boolean> = new EventEmitter<boolean>();
+        mobile = true;
+        mobileConfig: MobileModeConfig = {
             title: 'title',
             approveButtonText: 'approve',
             cancelButtonText: 'cancel',
             hasCloseButton: true
-        },
-        dialogDismiss: (backupArgument: string) => {
-        },
-        dialogApprove: () => {
-        },
-        openChange: new EventEmitter<boolean>()
-    };
+        };
+        dialogApprove(): void {
+            //
+        }
+        dialogDismiss(backup: string): void {
+            //
+        }
+        detectChanges(): void {
+            //
+        }
```

Comment:
I'm sorry, but why is this added?

Reply:
Particularly here it's just a stub for the specification.
But in general I added this method to the `ComboboxInterface` to be able to run it in another component

Reply:
Sorry, I meant the empty comments, assuming these are comments `//` :D I see before the blocks were empty.